docs/content/about.md

```diff
@@ -308,7 +308,7 @@ mvn clean install -DskipTests
 git checkout -b my_feature
 -- develop and commit
 ```
-![check_branch](/_static/fig/contribute_guidance/check_branch.png "check_branch")
+   ![check_branch](/_static/fig/contribute_guidance/check_branch.png "check_branch")
 5. Push your branch to your github
 ```bash
 git push origin my_feature
```

Comment:
add three spaces for bash in 3. 4. 5. since ``` will also lead to reorder.
   ```bash
   git push origin my_feature
   ```

Reply:
> add three spaces for bash in 3. 4. 5. since ``` will also lead to reorder.
> 
> ```shell
> git push origin my_feature
> ```

3. Clone and compile your Flink CDC project
  ```bash
git clone https://github.com/your_name/flink-cdc-connectors.git
cd flink-cdc-connectors
mvn clean install -DskipTests
  ```
4. Check to a new branch and start your work
  ```bash
git checkout -b my_feature
-- develop and commit
  ```
   ![check_branch](/_static/fig/contribute_guidance/check_branch.png "check_branch")
5. Push your branch to your github
  ```bash
git push origin my_feature
  ```

I can only add up to two spaces here in my local bash, adding three bash will be invalid.
Can you help me see if the document can be displayed normally?